swift_browser_ui/ui/server.py

```diff
@@ -65,6 +72,22 @@
 async def kill_sess_on_shutdown(app: aiohttp.web.Application) -> None:
     """Kill all open sessions and purge their data when killed."""
     logging.info(f"Gracefully shutting down the program at {time.ctime()}")
+
+    if setd.get("set_session_devmode", False):
```

Comment:
i understand the utility, however this leaves room for error in config given that this is relevant for login, i think we need another way to do this

Reply:
Yes, a dev server with auto reload would be better, and this would run only when running on the dev server, for example. I tried something on those lines, but didn't succeed yet. I'll try again some other time.

Reply:
yes, probably we can get inspired from https://github.com/CSCfi/metadata-submitter/pull/287